assets/component-slider.css

```
@@ -132,30 +108,20 @@ slider-component .slider-buttons {
 
 .slider-button {
   color: rgba(var(--color-foreground), 0.75);
-  border: 0.1rem solid rgba(var(--color-foreground), 0.08);
-  background-color: rgb(var(--color-background));
+  background: transparent;
+  border: none;
   cursor: pointer;
   width: 44px;
   height: 44px;
 }
 
 .slider-button:not([disabled]):hover {
   color: rgb(var(--color-foreground));
-  border-color: rgb(var(--color-foreground));
-  z-index: 1;
+  background-color: rgba(var(--color-foreground), 0.04);
 }
 
-.slider-button:first-of-type {
-  margin-right: -1px;
-}
-
-.slider-buttons--overlay .slider-button {
-  margin-top: -1px;
-  margin-bottom: -1px;
-}
-
-.slider-buttons--overlay .slider-button + .slider-button {
-  margin-right: -1px;
+.background-secondary .slider-button:not([disabled]):hover {
+  background-color: rgb(var(--color-background));
```

```suggestion
  background-color: rgb(var(--color-background), 0.04);
```
We can suggest the same opacity for the secondary background instead of opaque hover.